### ISSUE TITLE
Artwork: if album is not on Spotify

### ISSUE THREAD

**Issue by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#132]
If an album is not on Spotify
But we have it on a Bandcamp statement
Use the bandcamp `item_url` field to go and grab the artwork

**Comment by richardbirkin** [comment]
@g105b I am thinking of moving this to later down the line. It doesn't feel like a problem yet. Thoughts?

**Comment by g105b** [comment]
Yeah, that's a good call. It's not a necessary task to move forwards.
